### DISCUSSION TITLE
Filter by location

### DISCUSSION THREAD

**Discussion by tsgwiro1** [Donkie/Spoolman#491] [Q&A]
I use several dry boxes for all my active spools for printing and a separate location for all spools that are new / unopened as a sort of storage location. My normal workflow would be to select the spool that is used for the print in the Klipper WebUI (e.g. Mainsail) unfortunately all spools are listed including the ones in storage which should not be used for printing at this moment.

Would it be possible to predefine the sorting and/or filtering of the spools shown in Klipper so that all the active spools come first?
Would that be a feature that others would be interested in? Please comment.

**Comment by Donkie** [comment]
This is a feature request to your UI, e.g. Mainsail in this case. Spoolman already supports the functionality needed to achieve this.
